### PR TITLE
feat: initial setup for design tokens

### PR DESCRIPTION
**Pull request by ahmadshaheer**
Part of: https://github.com/SigNoz/engineering-pod/issues/1352

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "React Library",
```
Comment (YounixM) [inline tsconfig.json]: Should we rename this to SigNoz Design Tokens ?

Comment (YounixM) [reply tsconfig.json]: Very minor comment, can be taken up in next PR.

**Review by YounixM — DISMISSED** [review]

File: tsconfig.json (hunk 1 above)
Comment (ahmadshaheer) [reply tsconfig.json]: Done, @YounixM.

**Review by YounixM — DISMISSED** [review]

**Review by vikrantgupta25 — APPROVED** [review]

**Review by SagarRajput-7 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Comment by YounixM** [comment]
Merging the PR as it is blocked on CI. 

Can be fixed without blocking this.

**Pull request merged**